gammapy/utils/random.py

```diff
@@ -39,38 +40,32 @@ def sample_sphere(size, lon_range=None, lat_range=None, unit='rad'):
     ----------
     size : int
         Number of samples to generate
-    lon_range : tuple
-        Longitude range (min, max) tuple in range (0, 360)
-    lat_range : tuple
-        Latitude range (min, max) tuple in range (-90, 90)
-    units : {'rad', 'deg'}
-        Units of input range and returned angles
+    lon_range : `~astropy.units.Quantity`, optional
+        Longitude range (min, max) in range (0, 360) deg
```

Comment:
Would it be easy to make the code just work for longitudes in the range `(0, 360)` and `(-180, 180)`?
I.e. not have this requirement of `lon_range` being in the range `(0, 360)`?
If yes, please do.